### ISSUE TITLE
v.1.12.0 is not working

### ISSUE THREAD

**Issue by kostya93** [Vocab-Apps/anki-hyper-tts#193]
When I'm trying to listen Preview:

`Encountered an unknown error while Playing Sound Preview: 'str' object has no attribute 'items'`

![image](https://github.com/user-attachments/assets/acbbec72-c99a-4bb9-aa10-55dabbfbb824)

When I'm trying to run saved presents:

```
Encountered an unknown error while Opening HyperTTS Dialog from Browser: expected typing.Mapping, got str instead

Trace:
    $: TypeHint(111623432.voice.TtsVoiceId_v3)
    .voice_key: TypeHint(typing.Dict[str, typing.Any])
```

![Screenshot from 2024-08-11 17-31-10](https://github.com/user-attachments/assets/1db08543-89d6-4cdd-b0fd-416b6fb42119)

**Comment by Anatoool** [comment]
I have the same issue. I have the latest anki version.

**Comment by lndang1106** [comment]
Same problem 😢

**Comment by luc-vocab** [comment]
many apologies for the issue, I'm tracking down the bug, can you tell me which service / which voice you are using ?

**Comment by luc-vocab** [comment]
should be fixed in HyperTTS 1.12.4 or later.

**Comment by lndang1106** [comment]
> should be fixed in HyperTTS 1.12.4 or later.

Just updated to 1.12.5 and it seems to be fine now.

**Comment by luc-vocab** [comment]
thank you, please let me know if you run into any other issue

**Comment by Anatoool** [comment]
In my case it also was fixed. Thank you.

**Comment by luc-vocab** [comment]
should be fixed in HyperTTS 1.12.X
if anyone else hits this issue please email hypertts@airpost.net
